_schemaV2.graphql

```diff
@@ -4278,7 +4297,7 @@ type CommerceSubmitPendingOfferPayload {
 }
 
 type CommerceUser {
-  internalID: String!
+  id: String!
```

Comment:
@ashkan18  So these I’m not filtering out, as it would leave the type empty and then break things. However, as I understand it, these are only for stitching purposes and actually shouldn’t stay in the final public schema, is that correct?

Reply:
yes, this makes sense. We use the id, type field in stitching to resolve `buyer`, `seller` to a `user` or `partner`. These don't have to exist in the generated stitched schema.

Reply:
https://github.com/artsy/metaphysics/blob/e07150646f4230f9fc3786436deaf45838fa4693/src/lib/stitching/exchange/stitching.ts#L41